cpp/log/etcd_consistent_store.h

```diff
@@ -106,12 +103,12 @@ class EtcdConsistentStore : public ConsistentStore<Logged> {
 
   std::string GetUnsequencedPath(const std::string& hash) const;
 
-  std::string GetSequencedPath(int64_t seq) const;
-
   std::string GetNodePath(const std::string& node_id) const;
 
   std::string GetFullPath(const std::string& key) const;
 
+  void CheckMappingIsOrdered(const ct::SequenceMapping& mapping) const;
```

Comment:
Ah yes, it *was* like that originally, but I got a multiple definition error when linking and (for some weird reason) assuming it was because ${template}, but that's clearly bogus and upon closer inspection it's just because cluster_state_controller-inl.h was directly including this file.  Fixed that, fixed this.